ini/stm32f1.ini

```diff
@@ -282,3 +282,14 @@ build_unflags        = ${common_stm32.build_unflags} -DUSBCON -DUSBD_USE_CDC
 extra_scripts        = ${common_stm32.extra_scripts}
   pre:buildroot/share/PlatformIO/scripts/generic_create_variant.py
   buildroot/share/PlatformIO/scripts/stm32_bootloader.py
+
+#
+# Malyan M200 (STM32F103CB)
+#
+[env:STM32F103CB_malyan]
+platform    = ${common_stm32.platform}
+extends     = common_stm32
+board       = malyanm200_f103cb
```

Comment:
Which variant this board will use?

Most of the stm32duino variants can’t be used in marlin without minor changes.

Reply:
only used by BOARD_MALYAN_M200

Reply:
malyanm200_f103cb is part of platformio provided boards https://docs.platformio.org/en/latest/boards/ststm32/malyanm200_f103cb.html

Reply:
And the variant?

we need to make sure the pins in variant.h file are all compile-time constants. That is the main reason we copy variants to marlin.